server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationFileCreator.java

```diff
@@ -171,6 +173,8 @@ private void writeCleanUpScript() throws IOException {
         BashScript cleanupScript = new BashScript(cfnModule.getFileAccess(), FILENAME_CLEANUP);
         // Delete Bucket
         cleanupScript.append(CLI_COMMAND_DELETEBUCKET + cfnModule.getBucketName() + " " + CLI_PARAM_FORCE);
+        // Echo delete Stack
+        cleanupScript.append(COMMAND_ECHO + " " + STRING_DELETESTACK + " " + cfnModule.getStackName());
```

Comment:
The `COMMAND_ECHO` already contains a blank, do you need a additional one?

You could add to the `STRING_DELETESTACK` a blank as well.